docs/separating-entity-definition.md

```diff
@@ -112,6 +112,44 @@ export const PersonSchema = new EntitySchema({
 });
 ```
 
+Example with embedded entity:
+
+```ts
+import {EntitySchema} from "typeorm";
+
+const AddressEntity = new EntitySchema({
```

Comment:
since technically `entity` is an equivalent of a database `table`, it's better to call it `AddressEmbed` to avoid confusion.